core/shared/src/test/scala/fs2/StreamSuite.scala

```diff
@@ -427,176 +427,188 @@ class StreamSuite extends Fs2Suite {
       }
     }
 
-    group("map") {
-      property("map.toList == toList.map") {
-        forAll { (s: Stream[Pure, Int], f: Int => Int) =>
-          assert(s.map(f).toList == s.toList.map(f))
+    test("#2072 - stream canceled while resource acquisition is running".only) {
+      for {
+        ref <- Ref.of[IO, Boolean](false)
+        _ <- testCancelation {
+          // This will be canceled after a second, while the acquire is still running
+          Stream.bracket(IO.sleep(1100.millis))(_ => ref.set(true))
         }
-      }
-
-      test("regression #1335 - stack safety of map") {
-        case class Tree[A](label: A, subForest: Stream[Pure, Tree[A]]) {
-          def flatten: Stream[Pure, A] =
-            Stream(this.label) ++ this.subForest.flatMap(_.flatten)
-        }
-
-        def unfoldTree(seed: Int): Tree[Int] =
-          Tree(seed, Stream(seed + 1).map(unfoldTree))
-
-        assert(unfoldTree(1).flatten.take(10).toList == List.tabulate(10)(_ + 1))
-      }
+        _ <- IO.sleep(200.milliseconds)
+        released <- ref.get
+      } yield assert(released)
     }
+  }
```

Comment:
From here to the end is just the result of noticing the stream cancelation test group accidentally included tests unrelated to stream cancelation -- I moved the closing brace up to right point which then left shifted all the accidentally captured tests.

Reply:
It is all whitespace enough, so Github can hide it.